chat-plugins/info.js

```diff
@@ -1766,7 +1767,7 @@ var commands = exports.commands = {
 		}
 		if (target === 'all' || target === 'star' || target === 'player') {
 			matched = true;
-			buffer += '<a href="http://www.smogon.com/sim/faq#star">Why is there this star (&starf;) in front of my username?</a><br />';
+			buffer += '<a href=\"http://www.smogon.com/sim/faq#star\">Why is there this star (&starf;) in front of my username?</a><br />';
```

Comment:
Why are you adding forwardslashes on this? It's surrounded in single quotes, so they aren't needed.